frontend/src/constants/census.ts

```diff
@@ -2,14 +2,15 @@
  * This file was automatically generated.
  * DO NOT MODIFY IT BY HAND. Instead, run "yarn codegen" to regenerate it.
  */
+import { CONFIG } from '../config/config'
 
 type censusMapping = {
   [category: string]: string[]
 }
 
 export const CENSUS_MAPPING_ERROR = 'No Census Mapping Detected'
 
-export const CENSUS_MAPPING: censusMapping = {
+export let CENSUS_MAPPING: censusMapping = {
```

Comment:
Does this export the thing as expected even in the case below where the value is overridden?

Reply:
Yes sir